src/api/status/public/js/build-log/check-for-build.js

```diff
@@ -42,4 +43,5 @@ export default async function checkForBuild() {
       reader.read().then(processLog).catch(finish);
     }
   }
+  buildHeader(status);
```

Comment:
This needs to move up before the `if(build)`, so even when there is an ongoing build, the header will still be updated.